sphinx-book-theme is failing with sphinx 7

As seen in the CI failure on PR #540.

Error:
> Exception occurred:
>   File "/home/runner/micromamba/envs/geocat-examples/lib/python3.8/site-packages/pydata_sphinx_theme/bootstrap_html_translator.py", line 27, in visit_table
>     self.generate_targets_for_table(node)
> AttributeError: 'BootstrapHTML5Translator' object has no attribute 'generate_targets_for_table'
> The full traceback has been saved in /tmp/sphinx-err-spmyomgq.log, if you want to report the issue to the developers.
> Please also report this if it was a user error, so that a better error message can be provided next time.
> A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
> make: *** [Makefile:20: html] Error 2
> writing output... [  2%] gallery-geocat-comp/comp_only/sg_execution_times
> Error: Process completed with exit code 2.

See the CI failure here: https://github.com/NCAR/geocat-examples/actions/runs/6268815224/job/17024348024
Related issue on the theme repo here: https://github.com/executablebooks/sphinx-book-theme/issues/749

--- a/conda_environment.yml
+++ b/conda_environment.yml
@@ -17,7 +17,7 @@ dependencies:
   - geographiclib
   - jupyterlab
   - make
-  - matplotlib
+  - matplotlib<3.8
   - metpy
   - mock
   - nbsphinx

--- a/conda_environment.yml
+++ b/conda_environment.yml
@@ -15,7 +15,6 @@ dependencies:
   - holoviews
   - cartopy
   - geographiclib
-  - jupyter
   - jupyterlab
   - make
   - matplotlib
@@ -27,11 +26,9 @@ dependencies:
   - pint
   - pip
   - scikit-learn
-  - sphinx
   - sphinx-book-theme
   - sphinx-design
   - sphinx-gallery
-  - sphinx_rtd_theme
   - xarray<=2023.02.0
   - wrf-python
   - pip:

--- a/docs/_templates/breadcrumbs.html
+++ b/docs/_templates/breadcrumbs.html
@@ -1,4 +1,0 @@
-{%- extends "sphinx_rtd_theme/breadcrumbs.html" %}
-
-{% block breadcrumbs_aside %}
-{% endblock %}

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -90,7 +90,6 @@ warnings.filterwarnings("ignore",
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-import sphinx_rtd_theme
 
 html_theme = 'sphinx_book_theme'
 html_title = ""

